### PR TITLE
Anatomy

### PR DESCRIPTION
**Pull request by kambysese**
@Eric89GXL Picking up where I left of on issue #60. Please review.

### CODE REVIEW

File: bin/run_fs_recon.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                           'for this script')
+
+    if op.isdir(op.join(subjects_dir, subject)) and not force:
+        raise RuntimeError('%s FreeSurfer directory exists. '
+                           'Use command line option --force to overwrite '
+                           'previous reconstruction results.' % subject)
+    if force:
+        shutil.rmtree(op.join(subjects_dir, subject))
+
+    os.mkdir(op.join(subjects_dir, subject))
```
Comment (Eric89GXL) [inline bin/run_fs_recon.py]: I don't think you need this line, the next line should create it

File: bin/run_mne_bem.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    run_subprocess(['mne', 'make_scalp_surfaces', '--overwrite', '--subject', subject])
+    if op.isfile(op.join(subjects_dir, subject, 'bem/%s-head.fif' % subject)):
+        os.rename(op.join(subjects_dir, subject, 'bem/%s-head.fif' % subject),
+                  op.join(subjects_dir, subject, 'bem/%s-head-sparse.fif' % subject))
+    os.symlink((op.join(subjects_dir, subject, 'bem/%s-head-dense.fif' % subject)),
+               (op.join(subjects_dir, subject, 'bem/%s-head.fif' % subject)))
+
+    # Create source space
+    run_subprocess(['mne_setup_source_space', '--subject', subject, '--spacing', '%.0f' % 5, '--cps'],
+                   env=this_env)
```
Comment (Eric89GXL) [inline bin/run_mne_bem.py]: I don't think we actually use this source space

**Comment by Eric89GXL** [comment]
Other than my comments this looks reasonable to me. I assume you've been using it and it works?

File: bin/run_mne_bem.py (hunk 2 above)
Comment (kambysese) [inline bin/run_mne_bem.py]: What source space would you recommend? I've been using a 5mm spaced source space grid for my MEEG study and my fsaverage src space is a 5p7...

**Comment by kambysese** [comment]
Yes it has worked for me with both three and single layer BEMs across two different datasets. I've not tested it on the example ds recently.

**Comment by Eric89GXL** [comment]
It's alright, it's more important that it works on real data anyway. We can iterate in `master` if we need to.

**Comment by Eric89GXL** [comment]
Thanks for the enhancement @kambysese

**Pull request merged**